R/CreatePackageReport.R

```diff
@@ -17,7 +17,8 @@
 CreatePackageReport <- function(pkg_name
                                 , pkg_reporters = DefaultReporters()
                                 , pkg_path = NULL
-                                , report_path = file.path(getwd(), paste0(pkg_name, "_report.html"))
+                                , report_path = file.path(path.expand("~"),paste0(pkg_name, "_report.html"))
+                                , auto_open_report = TRUE
```

Comment:
I hate us adding a flag to the function signature. Especially for a minor convenience.

I think we should just make this the only behavior until someone complains that they want to be able to toggle it